README.md

```diff
@@ -1,13 +1,18 @@
 # Golden Sample iOS App
 
 ## Overview
-The golden sample is a public repository that contains samples, examples and best practices. This repository should be used internally and externally to show the Backbase approach, architecture and way of working to our developers and customers.
+This repository serves as a public reference point, showcasing various components, examples, and recommended practices employed within the Backbase development environment. Its intended audience encompasses both internal and external developers, offering insights into Backbase's development approach, architecture, and workflow.
 
-## Running and building
+## Prerequisites
+Building and running this project necessitates access to artifacts hosted in designated [Backbase repository](https://repo.backbase.com). Read permissions for these repositories are required.
```

Comment:
It would be nice if we can add a link for a guide on how to setup those credentials.

Reply:
✅